cprover_bindings/src/goto_program/typ.rs

```diff
@@ -41,6 +41,10 @@ pub enum Type {
     FlexibleArray { typ: Box<Type> },
     /// `float`
     Float,
+    /// `Half float`
+    Float16,
+    /// `float 128`
```

Comment:
I don't think left ticks make sense with those (elsewhere there is C syntax between those tick marks). You could fix that by using `_Float16` and `_Float128`, which are the type names that will be supported by the next C standard.